Automattic/wp-calypso
# P2: customize login page

#### Changes proposed in this Pull Request

Based #55746

* Customizes the login page for P2 flows, as described in p9lV3a-2Lp-p2. This is part of the P2 Onboarding project.

#### Testing instructions



* Regression test: verify that non-P2 flows have the regular login screen.
* Check that nothing looks broken for the following:
    * login screen when accepting P2 invites
    * sign up screen for the `/start/p2` flow
* Check that login works the same -- login errors are displayed as appropriate, login succeeds as appropriate.

#### Screens

**New P2 login screen (username/email step)**
<img width="800" alt="Screen Shot 2021-09-03 at 8 45 25 PM" src="https://user-images.githubusercontent.com/730823/132007424-a0300582-ba17-4d71-9f4a-f4a50938fafd.png">


**New P2 login screen (password step), at 80%**
<img width="1680" alt="Screen Shot 2021-09-03 at 8 45 46 PM" src="https://user-images.githubusercontent.com/730823/132007504-a00de52f-b296-4349-b3ae-83af63574218.png">
